app/src/DTO/LdapEntryDTO.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\DTO;
+
+use RuntimeException;
+use Symfony\Component\Ldap\Entry;
+
+class LdapEntryDTO
+{
+    //TODO Create attrribute associate with LdapEntry
+    //TODO Create getter and setter
+
+    /**
+     * @var string
+     */
+    public $distinguishedNames;
+
+    /**
+     * @var array
+     */
+    public $attributes;
+
+    public function setdn($distinguishedNames): void
+    {
+        $this->dn = $distinguishedNames;
+    }
```

Comment:
Makes no sense... 

1. Why plural ?
2. You created a `distinguishedNames` but set `dn` !